index.html

```diff
@@ -2,9 +2,12 @@
 <html lang="en">
```

Comment:
you have aproblem with structure of your project. You need to create the directory "about" and inside you must put file index.html
![image](https://user-images.githubusercontent.com/79144475/196561780-01ce8fc0-dc47-4d45-b062-b13c648fda8f.png)